lib/window.js

```diff
@@ -0,0 +1,2 @@
+// Simple helper variable to distinguish between browser and Node, and values respectively a "window" or a "global" object
+export const _window = typeof window !== 'undefined' && typeof window.document !== 'undefined' ? window : global;
```

Comment:
Let's add an `isBrowser` here, and make this a little more explicit so that it'll be easier to add logic for when this needs to support Deno in the future =D

```suggestion
const hasWindow =typeof window !== 'undefined';
const hasDocument = hasWindow && window.document;
const isBrowser = hasWindow && hasDocument;
const context = isBrowser ? window : global;

export { context, isBrowser };
```